spec/support/geocoder_stubs.rb

```diff
@@ -1,41 +0,0 @@
-Geocoder.configure(ip_lookup: :test)
-
```

Comment:
I don't see these stubs being used anywhere. Commits are from long ago. Maybe nice to confirm with Katherine (account team) that we are not overlooking anything? Or have another Joy engineer double check this.

Reply:
I think it should be safe because we have Geocoder test configuration set up elsewhere:

https://github.com/18F/identity-idp/blob/cd1f017c529d6cbe9fa4af9d8ded5684ccce3520/config/initializers/geocoder.rb#L25-L30

The only instance I spotted as potentially affected should work from that configured default stub:

https://github.com/18F/identity-idp/blob/cd1f017c529d6cbe9fa4af9d8ded5684ccce3520/spec/decorators/event_decorator_spec.rb#L29-L30

We probably want some better test coverage for our reverse geolookup behavior with new-device sign-in email notifications and account History page, but we can follow-up with that separately if we need additional stubs.

Reply:
Thanks, @aduth!

Reply:
Resolving comment! Thanks for checking in